fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVMaxTimestampSnapshot.java

```diff
@@ -25,17 +25,17 @@
  * so the update time is used instead of the version
  */
 public class MTMVMaxTimestampSnapshot implements MTMVSnapshotIf {
-    // partitionId corresponding to timestamp
-    // The reason why both timestamp and partitionId are stored is to avoid
+    // partitionName corresponding to timestamp
+    // The reason why both timestamp and partitionName are stored is to avoid
     // deleting the partition corresponding to timestamp
-    @SerializedName("p")
-    private long partitionId;
+    @SerializedName("pn")
+    private String partitionName;
```

Comment:
When we upgrade from earlier versions, this field will be null. will there be any issues?

Reply:
This will only result in the inability to find the snapshot information previously refreshed by hive. Currently, hive does not support automatic refreshing, so it has no significant impact